monitoring/grafana.py

```diff
@@ -15,21 +16,19 @@ def parse():
     try:
         f = open(log_path, "r")
         contents = f.read().split("\n \n")
-
         if '' in contents:
             contents.remove('')
-
         for pi_info in contents:
             pi = pi_info.split("\n")
-
             pi_ip = pi[0].split('@')[1]
-            time = int(float(pi[0].split(' -')[0]) * 1000)
+            ts = datetime.strptime(pi[0].split(' -')[0], '%a %b %d %H:%M:%S %Y')
+            time = ts.timestamp() * 1000
             try:
-                cpu_load = float(pi[3].replace("CPU load: ", "").replace(" ", ""))
-                ram = float(pi[4].replace("RAM usage: ", "").replace(" ", ""))
-                disk_usage = float(pi[5].replace("Disk usage: ", "").replace(" ", ""))
-                pids = int(pi[6].replace("# of PIDs: ", "").replace(" ", ""))
-                temp = float(pi[7].replace("Temperature: ", "").replace(" Celsius", "").replace(" ", ""))
+                cpu_load = float(pi[-5].replace("CPU load: ", "").replace(" ", ""))
```

Comment:
why are you using negative indices?

Reply:
When the pi's go over the threshold, a message would be added to the log that breaks the grafana reporting. With the negative indices, that problem doesn't happen anymore.

Reply:
put a comment in to explain.